tests/functional/platforms/02-host-to-platform-upgrade.t

```diff
@@ -30,14 +30,16 @@ run_fail "${TEST_NAME_BASE}-validate-fail" \
     cylc validate "flow2.cylc"
 
 # Ensure that you can validate suite
-run_ok "${TEST_NAME_BASE}-validate" \
+run_ok "${TEST_NAME_BASE}-run" \
     cylc validate "${SUITE_NAME}" \
         -s "CYLC_TEST_HOST='${CYLC_TEST_HOST}'" \
         -s CYLC_TEST_HOST_FQDN="'$(ssh "$CYLC_TEST_HOST" hostname -f)'"
 
 # Check that the cfgspec/suite.py has issued a warning about upgrades.
-grep_ok "\[upgradeable_cylc7_settings\]\[remote\]host = ${CYLC_TEST_HOST}"\
-    "${TEST_NAME_BASE}-validate.stderr"
+grep_ok "\[t1\]\[remote\]host = ${CYLC_TEST_HOST}"\
+    "${TEST_NAME_BASE}-run.stderr"
+grep_ok "\[t2\]\[remote\]host = ${CYLC_TEST_HOST}"\
```

Comment:
I have run this test locally. Log is reporting:  `"\[T2\]\[remote\]host = ${CYLC_TEST_HOST}"\`, rather than the `t1` in the test.

Reply:
I can confirm that I get this error too.

Reply:
Had a dig, because of the way these warnings are raised it cannot tell tasks from families so I've updated it to `T2`.